cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlow.qll

```diff
@@ -249,6 +254,8 @@ module Global<ConfigSig Config> implements GlobalFlowSig {
   private module C implements FullStateConfigSig {
     import DefaultState<Config>
     import Config
+
+    predicate isSinkWithAnyState(Node sink) { none() }
```

Comment:
This should be removed. The entire translation from no-state to state should be within `DefaultState`.

Reply:
Fixed in 9a0a4a3611e7001624908f3dd1044fa3b4febb60.